lib/action_policy/policy/execution_result.rb

```diff
@@ -26,6 +26,10 @@ def success?
       def fail?
         @value == false
       end
+
+      def message
+        raise I18nNotAvailableError
```

Comment:
We don't need to add this method stub. If there is no `i18n`, there is no method.

The same with `full_messages`

Reply:
Done. My idea was that if the user tries to call a method described in docs and gets `undefined method` he might be confused a bit.